elements/rh-pagination/rh-pagination-lightdom.css

```diff
@@ -17,24 +17,28 @@ rh-pagination li {
 
 rh-pagination li > a {
   text-decoration: none;
-  color: var(--rh-color-text-primary-on-light, #151515);
+  color: var(--_list-anchor-text-color, var(--rh-color-text-primary-on-light, #151515));
   font-weight: var(--rh-font-weight-body-text-medium, 500);
   outline: none;
   width: var(--_link-size);
   height: var(--_link-size);
   display: grid;
   place-content: center;
-  background: var(--rh-color-surface-lighter, #f2f2f2);
+  background: var(--_list-anchor-bg-color, var(--rh-color-surface-lighter, #f2f2f2));
   box-sizing: border-box;
 }
 
 rh-pagination li a[aria-current="page"] {
-  background: var(--rh-color-surface-lightest, #ffffff);
+  background: var(--_list-anchor-current-page-bg-color, var(--rh-color-surface-lightest, #ffffff));
   position: relative;
 
-  --_border-color: #b8bbbe;
+  --_border-color: var(--_list-anchor-current-page-border-color, #b8bbbe);
```

Comment:
should this be a token @marionnegp ?

Reply:
Addressed in [db56e8a](https://github.com/RedHat-UX/red-hat-design-system/pull/1552/commits/db56e8a00a0ff6f5e66a7f0747206b10882d8499).